File: sdkwork-app-sdk-typescript/src/types/stats-export-form.ts
/** 统计导出请求 */
export interface StatsExportForm {
  /** 报告类型 */
  reportType: string;
  /** 开始时间 */
  startDate?: string;
  /** 结束时间 */
  endDate?: string;
  /** 导出格式 */
  format?: string;
  /** 接收邮箱 */
  email?: string;
  /** 设备类型 */
  deviceType?: string;
  /** 操作系统类型 */
  osType?: string;
}

File: sdkwork-app-sdk-csharp/Models/StatsExportForm.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class StatsExportForm
    {
        public string? ReportType { get; set; }
        public string? StartDate { get; set; }
        public string? EndDate { get; set; }
        public string? Format { get; set; }
        public string? Email { get; set; }
        public string? DeviceType { get; set; }
        public string? OsType { get; set; }
    }
}
